modalsheet/src/main/java/eu/wewox/modalsheet/ModalSheet.kt

```diff
@@ -45,6 +47,7 @@ public fun <T> ModalSheet(
     data: T?,
     onDataChange: (T?) -> Unit,
     cancelable: Boolean = true,
+    onBack: (() -> Unit)? = { onDataChange(null) },
```

Comment:
nit: would name it something like `onSystemBack` or `onBackPressed`  to stay consistent with `Activity::onBackPressed`

Reply:
Actually I was inspired by `BackHandler` 😅 

https://developer.android.com/reference/kotlin/androidx/activity/compose/package-summary#BackHandler(kotlin.Boolean,kotlin.Function0)

Reply:
Yeah, but in context of  `BackHandler` it makes more sense than when you call `ModalSheet`. But I don't insist on it so feel free to resolve

Reply:
Oh, I see, makes sense 👍 

I will rename it to `onSystemBack`